Resources/Prototypes/Maps/packed.yml

```diff
@@ -47,6 +47,7 @@
             Detective: [ 1, 1 ]
             SecurityCadet: [ 2, 2 ]
             Lawyer: [ 1, 1 ]
+            PermaPrisoner: [ 1, 0 ] # HARMONY
```

Comment:
Let's take prisoner off of Packed at the moment due to it being low pop. A bit of a shame since Packed has a rather nice perma, but it can be brought back later.